app/src/main/java/com/cmput301f18t20/medicalphotorecord/Problem.java

```diff
@@ -3,15 +3,16 @@
 import android.location.Location;
 
 import java.util.ArrayList;
+import java.util.Date;
 
 public class Problem extends Entry implements Refreshable {
     protected String description;
     protected ArrayList<Record> records = new ArrayList<>();
 
-    Problem() {
-        super();
+    public Problem();
+    public Problem(String createdByUserID, String title) {
```

Comment:
I believe this would throw a NonNumericUserIDException.

Reply:
fixed